Sources/ExecutorType.swift

```diff
@@ -55,6 +83,12 @@ struct QueueExecutor: ExecutorType {
 
 }
 
+/// An operation queue manages a number of operation objects, making higher-
```

Comment:
Does this correctly hyphenate as `higher-level` in Xcode and Jazzy, or does it end up rendering as `higher- level` with a space in the middle due to the linebreak?